plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
+
+import time
+from utils.flask_server.base_flask_api_server import BaseAPIApplication
+
+class PDRPluginAPI(BaseAPIApplication):
+    '''
+    class PDRPluginAPI
+    '''
+
+    def __init__(self, isolation_mgr):
+        """
+        Initialize a new instance of the PDRPluginAPI class.
+        """
+        super(PDRPluginAPI, self).__init__()
+        self.isolation_mgr = isolation_mgr
+
+
+    def _get_routes(self):
+        """
+        Map URLs to function calls
+        """
+        return {
+            self.get_excluded_ports: dict(urls=["/excluded"], methods=["GET"]),
+            self.exclude_ports: dict(urls=["/excluded/<excluded_ports_str>"], methods=["PUT"]),
+            self.include_ports: dict(urls=["/excluded/<port_names_str>"], methods=["DELETE"])
```

Comment:
it is more correct to add these extra attributes in the body and not in the URL itself, please change also for the DELETE. also please change to a list of tuples [(port_name,ttl), (...,...), ... ], if there's no ttl add - (port_name,)

Reply:
Done

Reply:
Done